packages/react-devtools-shared/src/devtools/views/Profiler/SnapshotCommitListItem.js

```diff
@@ -44,6 +48,55 @@ function SnapshotCommitListItem({data: itemData, index, style}: Props) {
     selectCommitIndex,
   ]);
 
+  let dragStartCommit: DragStartCommit | null = null;
+  const maxCommitIndex = filteredCommitIndices.length - 1;
+
+  const handleDrag = (e: any) => {
+    if (e.buttons === 0) {
+      document.removeEventListener('mousemove', handleDrag);
+      const iframe = document.querySelector('iframe');
```

Comment:
If an iframe is present on the page, it will not handle `mousemove` event of document in which it is embedded. Because the most common case is 1 iframe, I did not added this fix for all the possible iframes, should I?